README.rst

```diff
@@ -3,9 +3,10 @@ mcinstall
 
 A script to quickly make/provision a fresh Miniconda installation from scratch.
 
-This will download a Miniconda binary from https://repo.continuum.io/miniconda/
-like https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh,
-unpack and install it locally. It was tested on macOS and Linux ok, and has
+This will download a Miniconda binary based on your operating system from https://repo.continuum.io/miniconda/
```

Comment:
I'm planning to add support for the existing (although no longer maintained) BerryConda distribution for ARM 6 and 7 which would not be downloaded from continuum.io. So I would not stress this so much here.

Reply:
ok. I will remove this and will only keep OS specific links for now.